packages/core-suggest/core-suggest.js

```diff
@@ -28,7 +28,7 @@ export default class CoreSuggest extends HTMLElement {
     this._xhr = new window.XMLHttpRequest()
     this.id = this.id || getUUID()
 
-    if (IS_IOS) this.input.setAttribute('role', 'combobox') // iOS does not inform about editability if combobox
+    this.input.setAttribute('role', 'combobox')
```

Comment:
I assume this was removed after testing :)

Reply:
Yep, `role="combobox"` seems to be more helpful across all platforms, so we saw little point in only using it for iOS.  Also resolves the disparity between the FAQ and what was actually in the code (albeit as a third option).